sdk/cognitivelanguage/azure-ai-language-conversations/samples/async/sample_authentication_async.py

```diff
@@ -0,0 +1,70 @@
+# coding=utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_authentication_asyncasync.py
```

Comment:
and in below for USAGE

```suggestion
FILE: sample_authentication_async.py
```